packages/browser/src/__tests__/__snapshots__/config-snapshot.test.ts.snap

```diff
@@ -182,6 +182,19 @@ exports[`config snapshot for PostHogConfig 1`] = `
       ]
     }
   ],
+  "disable_conversations": [
+    "false",
+    "true"
+  ],
+  "conversations": [
```

Comment:
ok, going through with a nitpicking mindset :)

we could have `conversations: false`, instead of `disable_conversations: true`  
  
but arguably `disable_conversations` is more clear